README.md

```diff
@@ -1 +1,25 @@
-# knotx-adapter-archetype
\ No newline at end of file
+![Knot.x logo](https://avatars2.githubusercontent.com/u/24564424?v=3&s=400)
+
+# Knot.x Extension Maven Archetype
+
+Use this archetype to generate a custon _Knot.x_ extension.
```

Comment:
I've been wondering about that and was leaning towards providing a single archetype containing an example of each type of extension one could provide:

- Adapter
  - Service
  - Action
- Knot
- Handlebars helper

What do you think?